features/features.go

```diff
@@ -610,4 +610,11 @@ var (
 					productScope(kubernetes).
 					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
+
+	FeatureShortCertRotation = newFeatureGate("ShortCertRotation").
```

Comment:
If it is meant to be a CI only feature, then can we rename it to make it obvious, something like `CIOnlyShortCertRotation`

Reply:
This feature will be tested in CI, but it may be used in manual testing. I don't mind renaming it though